### PR TITLE
proctl: add page

### PR DESCRIPTION
**Pull request by CreativeCodeCat**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by HeCodes2Much** [comment]
@sebastiaanspeck and @spageektti i have fixed the issues sorry about the slow response.

File: pages/linux/proctl.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# proctl
+
+> Manage, Switch Between templated Licenses, and set Languages.
+> More information: <https://github.com/HeCodes2Much/proctl>.
+
+- List available licenses:
+
+`proctl -list-licenses/-ll`
```
Comment (spageektti) [inline pages/linux/proctl.md:8]:
```suggestion
`proctl {{-ll|-list-licenses}}`
```
Please update all examples like that. See style guide.

**Review by spageektti — COMMENTED** [review]
Hi, thanks for fixing the errors, but there are a few more.

**Comment by HeCodes2Much** [comment]
@spageektti Done

File: pages/linux/proctl.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# proctl
+
+> Manage, Switch Between templated Licenses, and set Languages.
```
[the remark this answers is not in the document]
Comment (HeCodes2Much) [reply pages/linux/proctl.md]: manage projects Licenses and Languages

**Review by spageektti — APPROVED** [review]

File: pages/linux/proctl.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# proctl
+
+> Manage, switch between templated licenses, and set languages.
```
Comment (spageektti) [inline pages/linux/proctl.md:3]:
```suggestion
> Manage projects licenses and languages, switch between templated licenses.
```

File: pages/linux/proctl.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (HeCodes2Much) [reply pages/linux/proctl.md]: templates are the licenses within a config folder where they have the templates for a list of licenses and the project licenses are the license for said project.

**Pull request merged**